package.json

```diff
@@ -12,19 +12,26 @@
     "@storybook/addon-controls": "^6.3.4",
     "@storybook/addon-docs": "^6.3.4",
     "@storybook/server": "^6.3.4",
+    "autoprefixer": "^9",
     "jasmine-core": "~2.4.1",
     "karma": "~6.3.4",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coffee-preprocessor": "~1.0.1",
     "karma-jasmine": "~0.3.8",
+    "postcss": "^7",
+    "postcss-loader": "~3.0.0",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
     "webpack-dev-server": "^3.11.2"
   },
   "license": "AGPL-3.0",
   "dependencies": {
+    "@rails/activestorage": "^6.1.4",
+    "@rails/ujs": "^6.1.4",
```

Comment:
We're not currently using ActiveStorage or UJS, so I'm not sure we need these here...